src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000044> <http://purl.obolibrary.org/obo/ENVO_03000043>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000044> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00002149>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000044> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_00002200>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000045> (brine rejection process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Brine_rejection") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000045> "A material transformation process during which brine is formed due to the expulsion of salt during frazil formation or a seawater freezing process. Expelled salt accumulates in brine droplets, which remain in a liquid state due to their increased salinity.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000045> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000045> "brine rejection process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000045> <http://purl.obolibrary.org/obo/ENVO_03000043>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000045> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00002149>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000045> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_00003044>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000046> (frazil)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://nsidc.org/cryosphere/glossary/term/frazil") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000046> "Small needle-like ice crystals consisting of nearly pure fresh water, which are suspended in water.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000046> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000046> "frazil"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000046> <http://purl.obolibrary.org/obo/ENVO_01000846>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000047> (frazil formation)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://nsidc.org/cryosphere/glossary/term/frazil") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000047> "A material accumulation process in which frazil ice crystals merge, forming thin sheets of ice.")
```

Comment:
COMMENT PENDING~!!!!

@pbuttigieg you're right. I'll change the label and definition to:

frazil ice formation: [def](https://en.wikipedia.org/wiki/Frazil_ice#Formation) = 
> A material accumulation process in which frazil ice crystals form due to the freezing of open, turbulent, supercooled water.

So obviously we now need: sea ice formation process: [def](https://en.wikipedia.org/wiki/Sea_ice#Formation) = 
> 



 which frazil ice crystals merge, forming thin sheets of ice.

> A material accumulation process in which frazil ice crystals merge, forming thin sheets of ice.